src/cascadia/TerminalSettingsModel/KeyMapping.h

```diff
@@ -53,22 +53,22 @@ namespace winrt::TerminalApp::implementation
     {
         KeyMapping() = default;
 
-        TerminalApp::ActionAndArgs TryLookup(winrt::Microsoft::Terminal::TerminalControl::KeyChord const& chord) const;
+        Microsoft::Terminal::Settings::Model::ActionAndArgs TryLookup(winrt::Microsoft::Terminal::TerminalControl::KeyChord const& chord) const;

```

Comment:
you should have been able to use `ActionAndArgs` directly, so long as this header file could not see the `implementation` header file for AAA.